python/ray/data/tests/test_execution_optimizer.py

```diff
@@ -1078,6 +1079,18 @@ def test_from_torch_e2e(ray_start_regular_shared, enable_optimizer, tmp_path):
     _check_usage_record(["FromItems"])
 
 
+def test__blocks_to_input_buffer_op_name(
```

Comment:
seems extra space? `test__blocks` -> `test_blocks`

Reply:
The method name is `_blocks_to_input_buffer` with a `_` prefix, so I have it with a double `__`. Let me know if the typical convention is to condense it to a single `_` though.

Reply:
yeah let's always use single `_` in test name, thanks.